apps/src/templates/ProtectedStatefulDiv.jsx

```diff
@@ -41,7 +42,7 @@ class ProtectedStatefulDiv extends React.Component {
     // when using the storybook styleguide, we don't really need to protect
     // anything, and actually we want to unmount/remount stuff all the time
     // when the page is hot-reloaded
-    if (!IN_STORYBOOK) {
+    if (!IN_STORYBOOK && !this.props.canUnmount) {
```

Comment:
Pretty useful to be able to turn off this error when you are testing with RTL which renders a non-shallow representation of the component.